.github/workflows/ci.yml

```diff
@@ -0,0 +1,97 @@
+name: UniformsCI
+
+on:
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - '*.md'
+      - 'docs/**'
+      - '.gitignore'
+      - 'website/**'
+      - '.vscode/**'
+      - 'templates/**'
+      - '.github/*.yml'
+      - 'performance/**'
+      - 'packages/**/*md'
+      - '.github/ISSUE_TEMPLATE/**'
+      - '.github/workflows/build-website.yml'
+
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+    branches:
+      - master
+    paths-ignore:
+      - '*.md'
+      - 'docs/**'
+      - '.vscode/**'
+      - 'website/**'
+      - '.gitignore'
+      - 'templates/**'
+      - '.github/*.yml'
+      - 'performance/**'
+      - 'packages/**/*md'
+      - '.github/ISSUE_TEMPLATE/**'
+      - '.github/workflows/build-website.yml'
+
+  release:
+      types: [released, prereleased]
+
+  env:
+    NODE_OPTIONS: --max_old_space_size=4096
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    steps:
+    - name: Checkout master
+      uses: actions/checkout@v2
+    - name: Use Node.js
+      uses: actions/setup-node@v1
+      with:
+        node-version: 12.x
+    - name: Cache Dependencies
+      id: cache-dependencies
+      uses: actions/cache@master
+      with:
+        path: |
+          node_modules
+          */*/node_modules
+        key: ${{ runner.os }}-${{ hashFiles('**/package.json') }}
+    - name: Install dependencies
+      if: steps.cache-dependencies.outputs.cache-hit != 'true'
+      run: yarn
+    - name: Build
+      run: yarn build
+    - name: Lint
+      run: yarn lint
+    - name: Unit Test
+      run: yarn test
+
+  publishPackages:
```

Comment:
I guess this is getting his realy bad flow from graphback. 
There are usually two files one for build one for release.
Doing both in one file seems overly complex

Reply:
https://github.com/redhat-developer/app-services-operator/tree/main/.github/workflows 

example

Reply:
Wow that was quick. Haha this was still a work in progress

Reply:
Cool, I'll have a look at that and maybe split it into two separate workflows